js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -47,9 +50,13 @@ export const TickDetailsDialog = ({tickId, isOpen, instigationSelector, onClose}
       style={{width: '80vw', maxWidth: '1200px', minWidth: '600px'}}
     >
       <TickDetailsDialogImpl tickId={tickId} instigationSelector={instigationSelector} />
-      <DialogFooter>
-        <Button onClick={onClose}>Close</Button>
-      </DialogFooter>
+      {/* The logs table uses z-index for the column lines. Create a new stacking index
+      for the footer so that the lines don't sit above it. */}
+      <div style={{zIndex: 1}}>
```

Comment:
Pretty irritating to have to do this, but the logs scrolling table is a strange beast that I don't want to try to fight here.

Reply:
Yeah... I think this is the right move here though. 👍